homeassistant/components/unifi/controller.py

```diff
@@ -128,6 +130,22 @@ def signal_options_update(self):
         """Event specific per UniFi entry to signal new options."""
         return f"unifi-options-{CONTROLLER_ID.format(host=self.host, site=self.site)}"
 
+    def update_wireless_clients(self):
+        """Update set of known to be wireless clients."""
+        wireless_clients = set()
```

Comment:
If you prefer sets, just convert it to sets when you load the data and convert it to lists when you save it.

Reply:
This is just a local list to know which new wireless clients we see from last update